Demo/src/main/java/com/paypal/android/api/model/CardPaymentToken.kt

```diff
@@ -1,6 +1,6 @@
 package com.paypal.android.api.model
 
-data class PaymentToken(
+data class CardPaymentToken(
```

Comment:
Ended up creating two token types in the Demo app: `CardPaymentToken` and `PayPalPaymentToken`. They are slightly different. Could have subclassed here, but it's more flexible to keep our data hierarchy flat in the Demo app to make change easier in the future.

Reply:
Do you mean the payment source they return?

Reply:
Yeah CardPaymentToken has `cardLast4` and a few other properties that don't apply to `PayPalPaymentToken`.